Predicting the genome-wide variant effects of DNA has been an ambitious ongoing effort.It is imperative to develop a firm understanding of these effects, 
as they provide insight into how complex diseases develop, how certain mutations are caused, and various agricultural traits. Although there are models that are capable of making these predictions already, 
they are not as capable as they could be, partially due to software limitations, and also due to them not taking advantage of multi-sequence alignment (MSA).

Here, we propose a Transformer-based DNA language model that integrates MSA to enhance the prediction of genome-wide variant effects. The model will leverage large-scale genomic sequences for self- supervised learning to capture complex sequence patterns and evolutionary
conservation. We will validate our predictions using ClinVar pathogenic and benign variants, comparing our modelâ€™s performance against machine learning models DeepSEA and Enformer. We expect that combining this type of machine learning model with MSA will improve prediction accuracy when compared to other models. The implications of this research are huge, as a deeper understanding of genome- wide variant effects will be instrumental to the advancement of medicine, science, agriculture, and anthropology.
